package/test/test.js

```diff
@@ -34,6 +38,20 @@ fetch('./test/mode.txt')
                 },
             });
 
+            const updateEditorValueInLocalStorage = () => {
```

Comment:
might as well put this in the IIFE as well

Reply:
I think we should use lodash, but if it's an issue I would keep this in some util function.